middleware/authentication.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var config = require('../config');
+
+const methodToCatch = ['PUT', 'DELETE', 'POST'];
+
+module.exports = (req, res, next) => {
+    if (methodToCatch.indexOf(req.method) !== -1) {
```

Comment:
You have still failed your conditions, `next()` is not called when the access token is good.